Library/Homebrew/cleanup.rb

```diff
@@ -50,7 +50,7 @@ def self.cleanup_cache
           cleanup_path(path) { path.unlink }
           next
         end
-        if path.basename.to_s == "java_cache" && path.directory?
+        if %w[java_cache npm_cache].include?(path.basename.to_s) && path.directory?
```

Comment:
I'm fine with the proposed version. Maybe now or when this array gets even longer, we might want to move that into a separate variable (or constant) with an intention-revealing name. But no strong feelings here.